ipaclient/install/client.py

```diff
@@ -835,19 +838,24 @@ def configure_certmonger(
     subject = str(DN(('CN', hostname), subject_base))
     passwd_fname = os.path.join(paths.IPA_NSSDB_DIR, 'pwdfile.txt')
     try:
-        certmonger.request_cert(
+        certmonger.request_and_wait_for_cert(
             certpath=paths.IPA_NSSDB_DIR,
             storage='NSSDB',
             nickname='Local IPA host',
             subject=subject,
             dns=[hostname],
             principal=principal,
-            passwd_fname=passwd_fname
+            passwd_fname=passwd_fname,
+            resubmit_timeout=120,
+        )
+    except Exception as e:
```

Comment:
Could you please enhance this code? I do not like broad exceptions, especially when so many arguments are being passed.

Best thing to do would be to remove the broad exception here:
https://github.com/freeipa/freeipa/blob/199d50a4c8ac2dd96a8bca3af4a90e4a9c05adf9/ipalib/install/certmonger.py#L386-L388
and replace it with `RuntimeError, dbus.exceptions.DBusException`

Reply:
This is really the case of *Turn all errors into a ScriptError*. The code logs the exception and throws a new exception with the stack context of the inner exception intact. Also I don't want to make the ``ipaclient`` package directly depend on dbus package.

Reply:
Alright